README.md

```diff
@@ -2,6 +2,7 @@
 
 This tool generates public and private key pairs for bitcoin-sv (SPV-Wallet) applications. The generated keys are stored in separate files (xpub_key.txt and xprv_key.txt).
 
+[SPV Wallet GitBook](https://bsvblockchain.gitbook.io/docs)
```

Comment:
IMHO this link should be introduced with some text, e.g. what it is and what an user can find there.